e2e/node_similarity_test/test_node_list_cosine_p/test.yml

```diff
@@ -0,0 +1,13 @@
+query: >
+        MATCH (m {id: 2}), (n)
+        WHERE n.id < 2
+        WITH collect(n) as nodes, m
+        
+        CALL node_similarity.cosine(m, nodes) YIELD *
+          
+        RETURN node1.id as node1, node2.id as node2, similarity
+        
+
+exception: >
+        Incompatible lengths of given arguments
+      
```

Comment:
Name of this test is `test_list_cosine_p`. On the other side, there is a testing of exception?

Reply:
The name of this `test is test_node_list_cosine_p`. One argument is a node, the other is a list (which has a length greater than 1). For the pairwise mode, both need to be the same length. Since they're not, an exception occurs.